cmake/modules/AddSwift.cmake

```diff
@@ -359,7 +359,7 @@ function(_add_variant_link_flags)
     endif()
   elseif("${LFLAGS_SDK}" STREQUAL "ANDROID")
     list(APPEND result
-        "-ldl" "-llog" "-latomic"
+        "-ldl" "-llog" "-latomic" "-licudata" "-licui18n" "-licuuc"
```

Comment:
I'll try to copy those libraries to Android device, and let you know. But the Hello World used to work before without copying those libraries!